package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "azure-openapi-linter",
-  "version": "0.1.4",
-  "description": "Azure OpenAPI Linter",
+  "name": "@microsoft.azure/azure-openapi-validator",
```

Comment:
As per our offline conversation, this is not a stand alone package and AutoRest would depend on this package instead of other way around. So we agreed to name it as `@microsoft.azure/` and make it not discoverable independently.